challenges/react-native-ecosystem/01.md

```diff
@@ -43,6 +43,10 @@ Example:
 
 - [ ] Use [tubo console log shortcut](https://github.com/Chakroun-Anas/turbo-console-log) (`command` + `option` + `L` or `ctrl` + `alt` + `L`)
 
+- [ ] Install `Expo Tools` extension to have a list of shortcuts in your app.json
+
+- [ ] Check in app.json if the extension work
```

Comment:
Check in app.json if the extension work, you should have something like this:

animated `.gif`

hit `control` + `space` on the file to have the preview

Reply:
fix